src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -24,19 +37,74 @@ const formConfig = {
   defaultDefinitions: {
   },
   chapters: {
-    form: {
-      title: 'Form',
+    schoolInformation: {
+      title: 'School Information',
       pages: {
-        page1: {
-          path: 'form-page',
-          title: 'First Page',
+        schoolInformation: {
+          path: 'school-information',
+          title: 'School Information',
           uiSchema: {
+            school: {
+              name: {
```

Comment:
I think we'll have to call this property `facilityCode` since that's what the custom component will return. The should only submit an address or a facility code and use the `view:cannotFindSchool` field to determine which is submitted. @cehsu LMK what you think of this and how we could do this.